### ISSUE TITLE
Tax on coupons

### ISSUE THREAD

**Issue by jeking928** [lat9/edit_orders#61]
Per ZC thread:
https://www.zen-cart.com/showthread.php?223121-Order-total-does-not-add-up

When updating the order status, the order total is being recalculated resulting in the wrong order total

**Comment by syntaxerror** [comment]
Look at this https://github.com/lat9/edit_orders/issues/60

**Comment by jeking928** [comment]
I did see that and thought it might be the same issue but didn't want to assume it was and mess up that thread if it was a different cause.

**Comment by lat9** [comment]
Did you try ticking the "Reset totals on update?" box?

**Comment by jeking928** [comment]
I don't see such a box. This site is using 4.2.3

**Comment by lat9** [comment]
That feature was introduced in EO v4.3.0.  Could you try with that version and setting?

**Comment by jeking928** [comment]
That explains it. They don't use EO any longer. They prefer to use the order comments so there is an audit trail, so I'm afraid no need to upgrade.

**Comment by lat9** [comment]
OK, color me confused.  If they're not using EO, what is the issue?

**Comment by jeking928** [comment]
They had it the past, which is why it was installed. They been reporting for months that some order totals didn't add up. Every time I tested by placing identical orders, the totals were correct. After months of this, they finally noticed the problem occurred when they updating the order status to 'shipped'. Turns out they were on the EO page when updating the status rather than the default orders page.

What they do now is update the status from the default orders page and don't use EO. I reported this here in case it was a bug. If the "Reset totals on update?" does fix this issue, then I'd say you're all good.

Crystal clear now?
